### PR TITLE
Bugfix 2FA theme: buttons white

### PR DESCRIPTION
**Pull request by arcuslinus**
Before:
screenshot_20180909_111457

After:
screenshot_20180909_111418

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@CrazyEasy I've *never* seen that bug. Do you use any custom CSS or theming?

File: apps/theming/css/theming.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@
 		label,
 		p,
 		#alternative-logins legend {
-			color: $color-primary-text !important;
+			color: $color-primary-text;
```
Comment (skjnldsv) [inline apps/theming/css/theming.scss]: Please use the css variables :) 
https://github.com/nextcloud/server/blob/master/core/css/css-variables.scss
`var (--color-primary-text);`

File: apps/theming/css/theming.scss [hunk 2]
```diff
@@ -171,7 +171,7 @@ input.primary,
 	#body-login {
 
 		a, label, p {
-			color: $color-primary-text !important;
+			color: $color-primary-text;
```
Comment (skjnldsv) [inline apps/theming/css/theming.scss]: `var (--color-primary-text);`

**Review by skjnldsv — CHANGES_REQUESTED** [review]

**Comment by skjnldsv** [comment]
@CrazyEasy Thanks a lot for your fix :)
A small fixes are needed to ensure compatibility with the accessibility app and the new design functions ;)

:+1:

**Comment by CrazyEasy** [comment]
@ChristophWurst 
>  I've never seen that bug. Do you use any custom CSS or theming?

No I don't. It's just the normal installation of Nextcloud 14.

**Review by skjnldsv — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
@CrazyEasy Thanks a lot for your fix :rocket: 

Normally we require commits to be signed off. But for this it is fine.
Looking forward to your next PR!

**Pull request merged**